extensions/Worker.Extensions.ServiceBus/src/ServiceBusMessageActions.cs

```diff
@@ -0,0 +1,168 @@
+using System;
+using System.Collections.Generic;
+using System.Threading;
+using System.Threading.Tasks;
+using Azure.Messaging.ServiceBus;
+using Google.Protobuf.WellKnownTypes;
+using Microsoft.Azure.Functions.Worker.Converters;
+using Microsoft.Azure.ServiceBus.Grpc;
+
+namespace Microsoft.Azure.Functions.Worker
+{
+    [InputConverter(typeof(ServiceBusMessageActionsConverter))]
+    public class ServiceBusMessageActions
+    {
+        private readonly Settlement.SettlementClient _settlement;
+
+        internal ServiceBusMessageActions(Settlement.SettlementClient settlement)
+        {
+            _settlement = settlement;
+        }
+
+        ///<inheritdoc cref="ServiceBusReceiver.CompleteMessageAsync(ServiceBusReceivedMessage, CancellationToken)"/>
+        public virtual async Task CompleteMessageAsync(
+            ServiceBusReceivedMessage message,
+            CancellationToken cancellationToken = default)
+        {
+            await _settlement.CompleteAsync(new() { Locktoken = message.LockToken}, cancellationToken: cancellationToken);
```

Comment:
```suggestion
            await _settlement.CompleteAsync(new() { Locktoken = message.LockToken}, cancellationToken: cancellationToken );
```

Reply:
I think this suggestion throws the space in the wrong spot.  It would be the curly brace right after `message.LockToken` rather than the closing paren, no?

Reply:
Edited the suggestion.